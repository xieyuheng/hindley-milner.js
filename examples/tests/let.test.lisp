(lambda (x) x)

(let ((id (lambda (x) x)))
  id)

(let ((id (lambda (x) x)))
  (id id))
