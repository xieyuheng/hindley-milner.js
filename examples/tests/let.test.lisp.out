(the (nu (A) (-> A A)) (lambda (x) x))
(the (nu (A) (-> A A)) (lambda (x) x))
(the (nu (A) (-> A A)) (lambda (x) x))
